core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/Analysis.scala

```diff
@@ -390,7 +390,9 @@ class Analysis(apps: Seq[ApplicationInfo]) {
     val allRows = apps.flatMap { app =>
       app.sqlIdToInfo.map { case (sqlId, sqlCase) =>
         SQLDurationExecutorTimeProfileResult(app.index, app.appId, sqlId, sqlCase.duration,
-          sqlCase.hasDatasetOrRDD, app.appInfo.duration, sqlCase.problematic,
+          sqlCase.hasDatasetOrRDD,
+          if (app.appInfo != null) app.appInfo.duration else Option(0L),
```

Comment:
nit: Can we use map instead of `if else` in arguments?

```suggestion
         app.appInfo.map(_.duration).getOrElse(0L),
```

Reply:
addressed